libs/api_common/store_helper_internal.cpp

```diff
@@ -691,3 +693,55 @@ ebpf_store_load_global_helper_information(
     ebpf_free(helper_name);
     return result;
 }
+
+_Must_inspect_result_ ebpf_result_t
+ebpf_store_clear(_In_ const ebpf_registry_key_t root_key_path)
+{
+    ebpf_registry_key_t root_handle = {0};
+    ebpf_registry_key_t provider_handle = {0};
+    uint32_t status;
+    ebpf_result_t result = EBPF_FAILED;
```

Comment:
nit: Instead of returning `EBPF_FAILED` for every error, we should use `win32_error_code_to_ebpf_result()` to convert to the corresponding `ebpf_result_t`

Reply:
fixed